src/content/apis/entertainment/meme-generator.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Meme Generator API
+description: Meme Generator API.
```

Comment:
Can we please have description different from name.
In case the original API doesn't have anything, you can be creative on your own 🙏

Reply:
Okie